packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -149,6 +149,8 @@ class _FlutterPlatform extends PlatformPlugin {
     bool subprocessActive = false;
     bool controllerSinkClosed = false;
     try {
+      // Callback hookups' Futures are uninteresting.
+      // ignore: unawaited_futures
```

Comment:
this comment doesn't explain why we don't want to wait for the controller itself to be done.